cowprotocol/accounting/rewards/mainnet/mainnet_dashboard_query_2510345.sql

```diff
@@ -0,0 +1,206 @@
+with
+
+block_range as (
+    select * from "query_3333356(start_time='{{start_time}}',end_time='{{end_time}}')"
+),
+
+results_per_tx as (
+    select * from "query_3427730(start_time='{{start_time}}',end_time='{{end_time}}',cte_name='results_per_tx')"
+
+),
+
+solver_slippage as (
+    select
+        solver_address as solver,
+        sum(eth_slippage_wei) * 1.0 / pow(10, 18) as slippage
+    from results_per_tx
+    group by solver_address
+),
+
+named_results as (
+    select * from "query_1541516(end_time='{{end_time}}',vouch_cte_name='named_results')"
+),
+
+-- BEGIN SOLVER REWARDS
+batch_rewards as (
+    select  --noqa: ST06
+        rbr.block_deadline,
+        rbr.block_number,  -- Null here means the settlement did not occur.
+        from_hex(rbr.solver) as winning_solver,
+        from_hex(rbr.tx_hash) as tx_hash,
+        -- Unpacking the data
+        cast(cast(rbr.data.winning_score as varchar) as int256) as winning_score,  -- noqa: RF01
+        cast(cast(rbr.data.reference_score as varchar) as int256) as reference_score,  -- noqa: RF01
+        cast(cast(rbr.data.surplus as varchar) as int256) as surplus,  -- noqa: RF01
+        cast(cast(rbr.data.fee as varchar) as int256) as fee,  -- noqa: RF01
+        cast(cast(rbr.data.execution_cost as varchar) as int256) as execution_cost,  -- noqa: RF01
+        cast(cast(rbr.data.capped_payment as varchar) as int256) as capped_payment  -- noqa: RF01
```

Comment:
any reason to not remove half of the table?
```suggestion
        from_hex(rbr.solver) as winning_solver,
        cast(cast(rbr.data.capped_payment as varchar) as int256) as capped_payment  -- noqa: RF01
        cast(cast(rbr.data.fee as varchar) as int256) as fee,  -- noqa: RF01
        cast(cast(rbr.data.execution_cost as varchar) as int256) as execution_cost,  -- noqa: RF01
```
Those are the only entries used.

Reply:
Good point, done